internal/server/data/migrations_test.go

```diff
@@ -161,3 +161,26 @@ func TestMigration_AddKindToProvider(t *testing.T) {
 		})
 	}
 }
+
+func TestMigration_AddAuthURLAndScopesToProvider(t *testing.T) {
+	for _, driver := range dbDrivers(t) {
+		t.Run(driver.Name(), func(t *testing.T) {
+			db, err := newRawDB(driver)
+			assert.NilError(t, err)
+
+			loadSQL(t, db, "202206151027-"+driver.Name())
```

Comment:
The file is incorrect. Should be 202206281027

```suggestion
			loadSQL(t, db, "202206281027-"+driver.Name())
```